### PR TITLE
doc: Change "configuration tool" to "ACRN configurator tool"

### PR DESCRIPTION
**Pull request by amyreye**
- Change "configuration tool" to "ACRN configurator tool" to match the tool's UI
- Change "configuration toolset" to "ACRN configurator tool" in cases that clearly refer to the configurator and not the entire toolset, update cross-ref

Signed-off-by: Amy Reyes <amy.reyes@intel.com>

### CODE REVIEW

File: doc/tutorials/vuart_configuration.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ only works on a single function.
 
 Only two vUART configurations are added to the predefined scenarios,
 but you can customize the scenarios to enable more using the :ref:`ACRN
-configuration toolset <acrn_config_workflow>`.
+configurator tool <acrn_configurator_tool>`.
```
Comment (amyreye) [inline doc/tutorials/vuart_configuration.rst:16]: Starting at line 241, this file also refers to "configuration tools" (example: "Setup ACRN vUART Using Configuration Tools"). Should I change these to "ACRN configurator tool"?

File: doc/developer-guides/hld/hld-splitlock.rst [hunk 2]
```diff
@@ -133,5 +133,5 @@ Disable Split-Locked Access Detection
 If the CPU supports Split-locked Access detection, the ACRN hypervisor
 uses it to prevent any VM running with potential system performance
 impacting split-locked instructions.  This detection can be disabled
-(eventually by using the ACRN configuration tools) for customers not
+(eventually by using the ACRN configurator tool) for customers not
```
Comment (dbkinder) [inline doc/developer-guides/hld/hld-splitlock.rst:136]:
```suggestion
(by changing the :option:`hv.FEATURES.ENFORCE_TURNOFF_AC` setting in ACRN configurator tool) for customers not
```

File: doc/tutorials/enable_ivshmem.rst [hunk 3]
```diff
@@ -59,21 +60,23 @@ where
 
 -  ``shm_size`` - Shared memory size of selected ``shm_name``.
 
-There are two ways to insert above boot parameter for ``acrn-dm``
+There are two ways to insert the above boot parameter for ``acrn-dm``:
 
--  Manually edit launch script file, in this case, user shall ensure that both
-   ``shm_name`` and ``shm_size`` match with that are defined via configuration tool GUI.
+-  Manually edit the launch script file. In this case, ensure that both
```
Comment (dbkinder) [inline doc/tutorials/enable_ivshmem.rst:65]: We should check to see if this launch script option can be set in the configurator too (vs. manually editing the script).

File: doc/tutorials/vuart_configuration.rst (hunk 1 above)
Comment (dbkinder) [reply doc/tutorials/vuart_configuration.rst:16]: If we're making this name change, we should do it everywhere including titles of documents if the reference is to the configurator tool itself (and not a generic mention of configuration tools).

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/tutorials/vuart_configuration.rst (hunk 1 above)
Comment (amyreye) [reply doc/tutorials/vuart_configuration.rst:16]: @gvancuts @NanlinXie In this file, does "configuration tools" refer to only the configurator tool or other tools? 
@dbkinder Regarding your comment, I did a search for "configuration tool" and "configuration editor" which led me to edit these files. Can you provide an example of other places that should be changed?

Comment (dbkinder) [reply doc/tutorials/vuart_configuration.rst:16]: I think you've got them covered now.

**Pull request merged**